redhat-cop/infra-ansible
# switched handler to use ansible-tower-service restart for SSL installation

### What does this PR do?
This will add a new handler to restart the systemd managed NGINX reverse proxy to activate the new custom certs.

### How should this be tested?
Install a new tower instance with custom certificates, then ensure the new certs (not the localhost certs) are served automatically without any other service restarts or system reboots. I was able to verify with a simple `systemctl restart nginx` after installing tower with custom certs that this will resolve the issue.

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #[292](https://github.com/redhat-cop/infra-ansible/issues/292)

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
